server/src/Schemas/users.js

```diff
@@ -18,7 +13,7 @@ const UserSchema = new Schema({
   tagLine: String,
   email: String,
   teams: [{ type: ObjectId, ref: constants.teams }],
-  teamSpecificData: [TeamSpecificData],
+  teamSpecificData: [{ type: Object, default: {} }],
```

Comment:
what is this?

Reply:
Planning to remove this field in the coming PRs

Reply:
I was planning to remove this field in the coming PRs, and store `admins` in the team table itself. Reverted for now. Will change if needed.